avocado/core/runners/sysinfo.py

```diff
@@ -0,0 +1,188 @@
+import multiprocessing
+import os
+import time
+import traceback
+
+from avocado.core import nrunner
+from avocado.core.runners.utils import messages
+from avocado.utils import software_manager
+from avocado.utils import sysinfo as sysinfo_collectible
+
+
+class PreSysInfo:
+    """
+    Log different system properties before start event.
+
+    An event may be a job, a test, or any other event with a
+    beginning and end.
+    """
+    sysinfo_dir = os.path.join('sysinfo', 'pre')
+
+    def __init__(self, config, sysinfo_files, queue):
+        """
+        Set sysinfo collectibles.
+
+        :param config: avocado configuration
+        :type config: dict
+        :param sysinfo_files: dictionary with commands/tasks which should be
+                              performed during the sysinfo collection.
+        :type sysinfo_files: dict
+        :param queue: queue for the runner messages
+        :type queue: multiprocessing.SimpleQueue
+        """
+        self.config = config
+        self.queue = queue
+        self._installed_pkgs = None
+        packages_namespace = 'sysinfo.collect.installed_packages'
+        self.log_packages = self.config.get(packages_namespace, False)
```

Comment:
Is this returning `None` by some reason? Maybe we can drop the second argument here.